Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -84,6 +84,8 @@
     <string name="accessibility_close">"Schließen"</string>
     <!-- XACT: menu description for screen readers -->
     <string name="accessibility_logo">"Corona-Warn-App"</string>
+    <!-- XACT: button description for screen readers to be appended at the end of content without focusable subcontent that are explored by touch -->
+    <string name="accessibility_button">" Button"</string>
```

Comment:
please remove this space here and add it in the formatter

Reply:
To be added in later PR